gen/nvml/device.go

```diff
@@ -2073,6 +2101,23 @@ func (GpuInstance GpuInstance) GetComputeInstanceProfileInfo(Profile int, EngPro
 	return GpuInstanceGetComputeInstanceProfileInfo(GpuInstance, Profile, EngProfile)
 }
 
+// nvml.GpuInstanceGetComputeInstanceProfileInfoV()
+func GpuInstanceGetComputeInstanceProfileInfoV(GpuInstance GpuInstance, Profile int, EngProfile int, InfoV ComputeInstanceProfileInfoV) Return {
+	var Info *ComputeInstanceProfileInfo_v2
+	switch info := InfoV.(type) {
+	case *ComputeInstanceProfileInfo_v2:
+		info.Version = StructVersion.ComputeInstanceProfileInfo_v2()
+		Info = (*ComputeInstanceProfileInfo_v2)(unsafe.Pointer(info))
+	default:
+		return ERROR_INVALID_ARGUMENT
```

Comment:
Question: This is not an NVML error, should we include it here?

Reply:
It basically is though. To me it feels equivalent to them setting an invalid `version` in the struct they pass in.

Reply:
OK. That's fair enough.